pkg/services/virtual-manager/manager.go

```diff
@@ -367,7 +361,17 @@ func (vm *VirtualManager) process() {
 
 func (vm *VirtualManager) registerVDeviceWithContainerId(podUID, contID string) (*vcudaapi.VDeviceResponse, error) {
 	klog.V(2).Infof("UID: %s, cont: %s want to registration", podUID, contID)
-	baseDir := filepath.Clean(filepath.Join(vm.cfg.VirtualManagerPath, podUID))
+
+	resp := vm.responseManager.GetResp(podUID, contID)
```

Comment:
The `responseManager` uses the `container name` as the key, so I think it will not be found when the `container id` is passed in.